app/controllers/api/v1/filters/labware_type_filter.rb

```diff
@@ -0,0 +1,8 @@
+# See README.md for copyright details
+
+class Api::V1::Filters::LabwareTypeFilter
+  def self.filter(params)
+    labware_type = LabwareType.find_by(name: params[:type])
+    labware_type ? { labware_type_id: labware_type.id } : nil
```

Comment:
This won't work if you search for a labware type that doesn't exist, it will return all labware rather than none of them.